webapp/src/utils/lyraConfig.ts

```diff
@@ -38,7 +38,7 @@ export class LyraConfig {
 
   public getProjectConfigByPath(projectPath: string): LyraProjectConfig {
     const projectConfig = this.projects.find(
-      (project) => project.path === projectPath,
+      (project) => project.relativePath === projectPath
```

Comment:
`relativePath` is normalized, but `projectPath` is not. This will fail for a `projects.yaml` with multiple slashes or with dot elements.

normalization is probably the right thing to do, but then we have to do it in ServerConfig too.

Reply:
fixed now both are normlized

Reply:
Resolved on 30a65c9945179377cde9acbdf0a04b193c6a951d by normalizing ServerConfig too.